geom/geom/src/TGeoManager.cxx

```diff
@@ -1434,7 +1434,15 @@ void TGeoManager::CloseGeometry(Option_t *option)
    if (fNLevel<30) fNLevel = 100;
 
 //   BuildIdArray();
-   Voxelize("ALL");
+   if (!opt.Contains("v")) {
```

Comment:
The option name is a bit counter-intuitive as one would expect "v" to voxelize and not to disable it (notably used by default in the Export method), maybe use "nv". I would add a comment line explaining it in the code just before its use.

Reply:
changed